x/claim/keeper/claim.go

```diff
@@ -769,7 +769,7 @@ func (k Keeper) IncrementClaimedSoFar(ctx sdk.Context, identifier string, amount
 }
 
 // ResetClaimedSoFar resets ClaimedSoFar for a all airdrops
-func (k Keeper) ResetClaimedSoFar(ctx sdk.Context) error {
+func (k Keeper) ResetClaimedSoFar(ctx sdk.Context, airdropIdentifier string) error {
```

Comment:
Great catch - are there other places we're assuming airdrops are synced up?

Reply:
that was the only one I noticed - I'm much less familiar with this code though.

But I think this type of thing is only relevant when we do something epochly for all airdrops - and I think the Reset is the only thing that falls in that category